mw-webapp/src/component/table/Table.tsx

```diff
@@ -1,15 +1,25 @@
 import {useEffect, useState} from "react";
```

Comment:
You can don't add component to table. Task is just create component. If you will use it in other files - there will be more mistakes and conflicts